app/views/schools/cohorts/add_participants.html.erb

```diff
@@ -5,6 +5,8 @@
   href: schools_cohort_path(@cohort.display_name))
 %>
 
+<%= render SchoolRecruitedTransitionComponent.new(school_cohort: @school_cohort) %>
+
```

Comment:
Would it not be better to use a layout for this, like https://github.com/DFE-Digital/early-careers-framework/blob/develop/app/views/layouts/admin/base.html.erb ?

Reply:
Yeah, was thinking about this as well. I though adding a single component is not enough to warrant new layout. If I were to use a layout, I'd need to create new base controller as well and change all the controllers ancestry (which sort of make sense).

Aaaaand, I just realized that such a base controller already exists! :D Yup, doing it then!